### PR TITLE
Persian lang

### PR DESCRIPTION
**Pull request by hamedkhosrawi**

### CODE REVIEW

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 1]
```diff
@@ -1014,43 +1014,70 @@ public ArrayList<String> getSqlDocumentation() {
...
-        b.append(",('testcaseexecution','crbversion','','en','Cerberus Version','This is the version of the Cerberus Engine that executed the <code class=\\'doc-crbvvoca\\'>test case</code>.<br>This data has been created for traceability purpose as the behavious of Cerberus could varry from one version to another.',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:1016]: Why was that line removed ? Can you put it back ?

File: hs_err_pid14404.log
Comment (vertigo17) [inline hs_err_pid14404.log:1]: Maybe you should exclude all .log file to git.
I will see if we can had that to avoid that comit.

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 2]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
         b.append(",('page_header','menuAccount','','en','Account','',NULL)");
         b.append(",('page_header','menuAccount','','fr','Compte','',NULL)");
+        b.append(",('page_header','menuAccount','','fa','حساب کاربری','',NULL)");
         b.append(",('page_header','menuAdmin','','en','Administration','',NULL)");
         b.append(",('page_header','menuAdmin','','fr','Administration','',NULL)");
+        b.append(",('page_header','menuAdmin','','fa','مدیریت','',NULL)");
         b.append(",('page_header','menuApplicationObjects','','en','Application Object','',NULL)");
         b.append(",('page_header','menuApplicationObjects','','ru','Объект приложения','',NULL)");
         b.append(",('page_header','menuApplicationObjects','','fr','Objet d\\'application','',NULL)");
+        b.append(",('page_header','menuApplicationObjects','','en','شیِ برنامه','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2017]: Should be fa.

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 3]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
+        b.append(",('page_header','menuReportingExecutionByTag','','fa','گزارش اجرای کمپین','',NULL)");
         b.append(",('page_header','menuReportingExecutionOverTime','','en','Execution Report Over Time','',NULL)");
         b.append(",('page_header','menuReportingExecutionOverTime','','fr','Rapport d\\'Execution dans le temps','',NULL)");
+        b.append(",('page_header','menuReportingExecutionOverTime','','fa','گزارش در طول زمان اجرا','',NULL)");
         b.append(",('page_header','menuReportingCampaignOverTime','','en','Campaign Report Over Time','',NULL)");
         b.append(",('page_header','menuReportingCampaignOverTime','','fr','Rapport de Campagne dans le temps','',NULL)");
+        b.append(",('page_header','menuReportingCampaignOverTime','','fa','گزارش در طول زمان کمپین','',NULL)");
         b.append(",('page_header','menuReportingExecutionList','','en','Execution List','',NULL)");
         b.append(",('page_header','menuReportingExecutionList','','fr','Liste d\\'Execution','',NULL)");
+        b.append(",('page_header','menuReportingExecutionList','','en','فهرست اجرا','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2108]: Should be fa

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 4]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
         b.append(",('page_header','menuReportingExecutionThreadMonitoring','','en','Cerberus Monitoring','',NULL)");
         b.append(",('page_header','menuReportingExecutionThreadMonitoring','','fr','Monitoring Cerberus','',NULL)");
+        b.append(",('page_header','menuReportingExecutionThreadMonitoring','','en','پایش سربروس','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2111]: should be fa

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 5]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
         b.append(",('page_header','menuRobot','','en','Robot','',NULL)");
         b.append(",('page_header','menuRobot','','fr','Robot','',NULL)");
+        b.append(",('page_header','menuRobot','','fa','ربات','',NULL)");
         b.append(",('page_header','menuRun','','en','Run','',NULL)");
         b.append(",('page_header','menuRun','','fr','Executer','',NULL)");
+        b.append(",('page_header','menuRun','','fa','اجرا','',NULL)");
         b.append(",('page_header','menuRunTestCase','','en','Run Test Case','',NULL)");
         b.append(",('page_header','menuRunTestCase','','fr','Executer un Cas de Test','',NULL)");
+        b.append(",('page_header','menuRunTestCase','','en','اجرای مورد آزمون','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2120]: should be fa

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 6]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
+        b.append(",('page_header','menuTest','','fa','آزمون','',NULL)");
         b.append(",('page_header','menuTestBattery','','en','TestCase Battery','',NULL)");
         b.append(",('page_header','menuTestBattery','','fr','Batterie de Cas de Test','',NULL)");
+        b.append(",('page_header','menuTestBattery','','fa','باتری مورد آزمون','',NULL)");
         b.append(",('page_header','menuTestCampaign','','en','Campaign','',NULL)");
         b.append(",('page_header','menuTestCampaign','','fr','Campagne de Test','',NULL)");
+        b.append(",('page_header','menuTestCampaign','','fa','کمپین','',NULL)");
         b.append(",('page_header','menuTestCase','','en','TestCase','',NULL)");
         b.append(",('page_header','menuTestCase','','fr','Cas De Test','',NULL)");
+        b.append(",('page_header','menuTestCase','','en','مورد آزمون','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2138]: should be fa

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 7]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
+        b.append(",('page_label','btn_create','','fa','ایجاد برچسب','',NULL)");
         b.append(",('page_label','btn_delete','','en','Delete Label','',NULL)");
         b.append(",('page_label','btn_delete','','fr','Effacer le Label','',NULL)");
+        b.append(",('page_label','btn_delete','','fa','حذف برچسب','',NULL)");
         b.append(",('page_label','btn_edit','','en','Edit Label','',NULL)");
         b.append(",('page_label','btn_edit','','fr','Editer le Label','',NULL)");
+        b.append(",('page_label','btn_edit','','fa','ویرایش برچسب','',NULL)");
         b.append(",('page_label','btn_view','','en','View Label','',NULL)");
         b.append(",('page_label','btn_view','','fr','Voir le Label','',NULL)");
+        b.append(",('page_label','btn_view','','en','مشاهده برچسب','',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2273]: should be fa

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java (hunk 1 above)
Comment (hamedkhosrawi) [reply source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:1016]: hi dear vertigo, maybe when i highlighted the line cause the deletion. because i copy the line to bottom then i translate to persian, i apologize

File: hs_err_pid14404.log
Comment (hamedkhosrawi) [reply hs_err_pid14404.log:1]: i add the .log to gitignore

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 8]
```diff
@@ -1814,327 +1814,463 @@ public ArrayList<String> getSqlDocumentation() {
...
         b.append(",('page_global','tab_definition','','fr','Définition','',NULL)");
+        b.append(",('page_global','tab_definition','','fa','تعریف','',NULL)");
         b.append(",('page_global','tab_advanced','','en','Advanced','',NULL)");
         b.append(",('page_global','tab_advanced','','ru','Детально','',NULL)");
         b.append(",('page_global','tab_advanced','','fr','Avancé','',NULL)");
+        b.append(",('page_global','tab_advanced','','fa','پیشرفته','',NULL)");
         b.append(",('page_global','traca','','en','Traceability','',NULL)");
         b.append(",('page_global','traca','','ru','Прослеживаемость','',NULL)");
         b.append(",('page_global','traca','','fr','Traçabilité','',NULL)");
+        b.append(",('page_global','traca','','en','قابلیت ردیابی','',NULL)");
```
[the remark this answers is not in the document]
Comment (hamedkhosrawi) [reply source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2040]: yes, sometimes unintentionally miss the fa insertion

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java (hunk 5 above)
Comment (hamedkhosrawi) [reply source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:2156]: fixed

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java (hunk 1 above)
Comment (hamedkhosrawi) [reply source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:1016]: fixed

File: source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java [hunk 9]
```diff
@@ -1115,48 +1115,73 @@ public ArrayList<String> getSqlDocumentation() {
...
         b.append(",('testcaseexecutionwwwsum','img_size_max','','en','Img_size_max','Size of the biggest Picture dowloaded during the scenario',NULL)");
         b.append(",('testcaseexecutionwwwsum','img_size_max','','fr','Img_taille_max',NULL,NULL)");
+        b.append(",('testcaseexecutionwwwsum','img_size_max','','fa','حداکثرـاندازهـتصویر','اندازه بزرگترین عکس دانلود شده در طول سناریو',NULL)");
         b.append(",('testcaseexecutionwwwsum','img_size_tot','','en','Img_size_tot','Total size of the Pictures for the scenario',NULL)");
         b.append(",('testcaseexecutionwwwsum','img_size_tot','','fr','Img_taille_tot',NULL,NULL)");
+        b.append(",('testcaseexecutionwwwsum','img_size_tot','','fa','مجموعـاندازهـتصویر','اندازه کل تصاویر برای سناریو',NULL)");
         b.append(",('testcaseexecutionwwwsum','img_tps','','en','Img_tps','Cumulated time for downloaded pictures for all the scenario',NULL)");
         b.append(",('testcaseexecutionwwwsum','img_tps','','fr','Img_tps',NULL,NULL)");
+        b.append(",('testcaseexecutionwwwsum','img_tps','','en','Img_tps','زمان انباشته شده برای تصاویر دانلود شده برای همه سناریوها',NULL)");
```
Comment (vertigo17) [inline source/src/main/java/org/cerberus/core/database/DocumentationDatabaseService.java:1135]: should be fa
